content/docs/0.8.x/continuous_delivery/multi_stage/index.md

```diff
@@ -5,26 +5,26 @@ weight: 1
 keywords: [0.8.x-cd]
 ---
 
-The definition of a multi-stage delivery manifests in a so-called **shipyard**. It can hold multiple stages with dedicated and opinionated delivery tasks to execute. Following this declarative approach, there is no need to write imperative pipeline code. Keptn takes the shipyard file and creates a task sequence for multi-stage delivery.
+The definition of a multi-stage delivery manifests in a so-called **shipyard**. It can hold multiple stages with dedicated and opinionated delivery tasks to execute. Following this declarative approach, there is no need to write imperative pipeline code. Keptn takes the shipyard  and creates a task sequence for multi-stage delivery.
 
 ## Declare Shipyard (before creating a project)
 
-The `shipyard.yml` file defines the stages each deployment has to go through until it is released in production. A shipyard file can consist of any number of stages with at least the name property. Besides, each stage can consist of sequence tasks (strategies) from the following list:
+* A shipyard is defined at the level of a project. This means that all services in a project share the same shipyard configuration. 
 
-* approval_strategy
-* deployment_strategy
-* test_strategy
+* A shipyard defines the stages each deployment has to go through until it is released in the final stage, e.g., the production stage. 
 
-A shipyard is defined at the level of a project. This means that all services in a project share the same shipyard configuration. 
+* A shipyard can consist of any number of stages with at least the name property. 
 
-### Name 
+* A stage can consist of task sequences.
 
-The name of the stage. This name will be used for the branch in the Git repository and Kubernetes namespace to which services at this stage will be deployed to. 
+### Definition of Stage 
+
+A stage is declared by its name. This name will be used for the branch in the Git repository and Kubernetes namespace to which services at this stage will be deployed to. 
```

Comment:
Yes, that is still true. Because the helm-service will create the namespace based on the stage name + service name and it does not matter where the helm-service is running.